scr/main/java/bittercode/util/DatabaseConfig.java

```diff
@@ -19,6 +19,19 @@ class DatabaseConfig {
         }
     }
 
+    try  {
+        input.close();
+    } catch (IOException e) {
+        e.printStackTrace();
+    }
+
+    try   {
+        input.close();
+    } catch (IOException e) {
+        e.printStackTrace();
+    }

```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider removing duplicate stream close blocks</b></div>
<div id="fix">

Consider removing the duplicate `input.close()` blocks. The stream is being closed twice which is unnecessary and could potentially cause issues.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">

```
 @@ -15,21 +15,13 @@
  try {
      prop.load(input);
 +    input.close();
  } catch (IOException e) {
      e.printStackTrace();
  }
 -}
 -
 -try  {
 -    input.close();
 -} catch (IOException e) {
 -    e.printStackTrace();
 -}
 -
 -try   {
 -    input.close();
 -} catch (IOException e) {
 -    e.printStackTrace();
 -}
 -}
```




```
 @@ -15,21 +15,13 @@
  try {
      prop.load(input);
 +    input.close();
  } catch (IOException e) {
      e.printStackTrace();
  }
 -}
 -
 -try  {
 -    input.close();
 -} catch (IOException e) {
 -    e.printStackTrace();
 -}
 -
 -try   {
 -    input.close();
 -} catch (IOException e) {
 -    e.printStackTrace();
 -}
 -}
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/8#issuecomment-2527744697>#4874c2</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged